ext/kv/README.md

```diff
@@ -1,3 +1,132 @@
 # deno_kv
 
-This crate provides a key/value store for Deno.
+This crate provides a key/value store for Deno. For an overview of Deno KV,
+please read the [manual](https://deno.land/manual/runtime/kv).
+
+## Storage Backends
+
+Deno KV has a pluggable storage interface that supports multiple backends:
+
+- SQLite - backed by a local SQLite database. This backend is suitable for
+  development and is the default when running locally.
```

Comment:
FYI people have been asking about alternative local backends: https://github.com/denoland/deno/discussions/18975 can this be addressed somehow?

Reply:
Added a note on additional backends